targets/python/source/source/PlayFabHTTP.py

```diff
@@ -20,28 +18,32 @@ class PlayFabRequestCommon(PlayFabBaseObject):
 class PlayFabResultCommon(PlayFabBaseObject):
     pass
 
-class PlayFabJsonError(Enum):
+class PlayFabJsonError():
     def __init__(self):
-        code = 0
-        status = ""
-        error = ""
-        errorCode = 0
-        errorMessage = ""
-        errorDetails = None
-
-class PlayFabJsonSuccess(Enum):
+        self.code = 0
+        self.status = ""
+        self.error = ""
+        self.errorCode = 0
+        self.errorMessage = ""
+        self.errorDetails = None
+
+class PlayFabJsonSuccess():
     def __init__(self):
-        code = 0
-        status = ""
-        data = None
+        self.code = 0
+        self.status = ""
+        self.data = None
 
 def DoPost(urlPath, request, authType, authKey, extraHeaders):
     if PlayFabSettings.TitleId == None:
         raise Exception("You must set your titleId before making an api call")
 
     url = PlayFabSettings.GetURL(urlPath)
 
-    j = json.dumps(request)
+    try:
+        j = json.dumps(request)
+    except v:
+        err = PlayFabJsonError().errorMessage = "{}{}".format(v , request)
```

Comment:
I don't think err is assigned to what you think it is here.
If it is, it's certainly not a clear line of code.